scalanativelib/scalanativebridges/0.3/src/mill/scalanativelib/bridge/ScalaNativeBridge.scala

```diff
@@ -0,0 +1,71 @@
+package mill.scalanativelib.bridge
+
+import java.lang.System.{err, out}
+
+import scala.scalanative.build.{Config, GC, Logger, Mode, Discover, Build}
+import ammonite.ops.Path
+import mill.scalanativelib.NativeConfig
+
+class ScalaNativeBridge extends mill.scalanativelib.ScalaNativeBridge {
+  val logger =
+    Logger(debugFn = msg => Unit, //err.println(msg),
+      infoFn  = msg => out.println(msg),
+      warnFn  = msg => out.println(msg),
+      errorFn = msg => err.println(msg))
```

Comment:
It would be nice to have some way to trigger debug messages being shown or not, maybe add a `nativeVerbose: Boolean` as a extra setting to control this. @lihaoyi is this the idiomatic way to do logging in mill?

Reply:
I agree it. It is currently way to verbose. I'm not really sure what the best way to handle this is. Happy to take suggestions @lihaoyi

Reply:
We could define any flags or configuraiton on `ScalaNativeModule` and plumb it down here and where-ever necessary. Then people can override it to configure